src/views/Correction.vue

```diff
@@ -179,7 +172,19 @@ export default class Correction extends Mixins(DateMixin, FilingTemplateMixin, L
 
     // try to fetch data
     try {
-      if (this.correctionId) {
+      // set current entity type
+      this.setEntityType(EntityTypes.BCOMP)
+
+      // initialize Fee Summary data
+      this.emitFilingData([{
```

Comment:
FYI (future TODO): we need to set/clear the filing data according to whether we have changes.

Reply:
Copy that, i will add a comment.

Reply:
Or... make it work?

Reply:
I'm OK either way.

Reply:
Absolutely, will do!

Reply:
For sake of expediency, added a TODO here. Will be applicable when implementing Alterations and beyond.